FireboltDotNetSdk.Tests/Integration/ExecuteTest.cs

```diff
@@ -449,20 +460,149 @@ public void CreateDropFillTable()
                         Assert.That(record.GetDateTime(8), Is.EqualTo(TypesConverter.ParseDateTime("2023-10-18 17:30:20.123456")));
                         Assert.That(record.GetDateTime(9), Is.EqualTo(TypesConverter.ParseDateTime("2023-10-18 17:30:20.123456+02")));
                         Assert.That(record.GetBoolean(10), Is.EqualTo(true));
+                        // byte array
+                        Assert.That(record.GetValue(11), Is.EqualTo(bytes));
+                        byte[] buffer = new byte[bytes.Length];
+                        long length = record.GetBytes(11, 0, buffer, 0, buffer.Length);
+                        Assert.That(length, Is.EqualTo(buffer.Length));
+                        Assert.That(buffer, Is.EqualTo(bytes));
                     }
                 }
 
                 Assert.True(selectOneRow(conn, 1));
                 Assert.False(selectOneRow(conn, 0));
 
-                CreateCommand(conn, DROP_TABLE).ExecuteNonQuery();
+                CreateCommand(conn, DROP_SIMPLE_TABLE).ExecuteNonQuery();
+            }
+        }
+
+        [Test]
+        public void CreateDropFillTableWithEmptyArrays()
+        {
+            CreateDropFillTableWithArrays("int", new int[0], typeof(int[]), "int", new int[0][], typeof(int[][]), "int", new int[0][][], typeof(int[][][]));
+        }
+
+        [Test]
+        public void CreateDropFillTableWithNullArrays()
+        {
+            CreateDropFillTableWithArrays("int", null, typeof(int[]), "int", null, typeof(int[][]), "int", null, typeof(int[][][]));
+        }
+
+        [Test]
+        public void CreateDropFillTableWithSingetonArrays()
+        {
+            CreateDropFillTableWithArrays(
+                "int", new int[] { 1 }, typeof(int[]),
+                "int", new int[][] { new int[] { 22 } }, typeof(int[][]),
+                "int", new int[][][] { new int[][] { new int[] { 333 } } }, typeof(int[][][]));
+        }
+
+        [Test]
+        public void CreateDropFillTableWithArrays()
```

Comment:
nit: Test has the same name as a function that's being used `CreateDropFillTableWithArrays`. Please rename, it's a bin confusing right now